.github/workflows/test-gb-25.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           - 'main'
           # - '0123456789abcdef0123456789abcdef01234567'
         reactant_commit:
-          - 'main'
+          - '33a3d346572391b31749b172636a38eeb15593dc'
```

Comment:
This can be reverted after https://github.com/EnzymeAD/Reactant.jl/pull/1557 is merged.